tests/test.cpp

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <stdint.h>
```

Comment:
needed for?

Reply:
Complaining about `uint8_t` being transitively included. It's not strictly needed,but removes a warning from my buid.